### PR TITLE
Add an option to override laterality test for JTT

### PR DESCRIPTION
**Pull request by mslw**
Added a switch (turned off by default) which disables laterality checking for JTT.

@christian-monch the JS part does what I would expect it to do, but I wasn't able to verify the server part - my intuition is that the checkbox shouldn't produce any additional data that gets passed to the server (and json file stored by DataLad) but that might be wrong, could you check it?

screenshot_JTT

### CODE REVIEW

File: entry.html
Comment (christian-monch) [inline entry.html]: It should read: "Lateralitätsprüfung aufheben"

**Review by christian-monch — CHANGES_REQUESTED** [review]
LGTM, there is just a small typo. Should be good to merge, once that is fixed.

File: entry.html
Comment (mslw) [reply entry.html]: Heh, there shows my German knowledge :laughing: Corrected.

**Comment by mslw** [comment]
Fixed the wording. Before merging let's maybe wait for confirmation from @VeronikaWu whether that matches her expectations, and whether only JTT needs this override.

**Comment by VeronikaWu** [comment]
This looks great. Until now, we need it only for JTT, but it possibly could occur for the other tests with laterality check as well. Depending on what would be easier to implement, the same check boxes or one checkbox for all affected tests would be great (but not urgent). Thank you so much for your quick help.

**Comment by mslw** [comment]
Thank you for the feedback, @VeronikaWu. I applied the same logic to other tests (form sections) that were check for laterality. So now each of these gets a slider that disables the laterality check for this test:
- Motorische Testung: Basisfähigkeiten
- Purdue Pegboard Test
- Jebsen Taylor Hand Function Test (JTT)
- Action Research Arm Test (ARAT)

@christian-monch I think this is ready to merge and deploy, although as always would be good if you checked for mistakes.

**Review by christian-monch — APPROVED** [review]

**Pull request merged**
